ajax/libs/popper.js/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "popper.js",
+  "filename": "popper.min.js",
+  "version": "0.5.0",
+  "description": "A kickass library to manage your poppers",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/FezVrasta/popper.js.git"
+  },
+  "keywords": [
+    "tooltips",
+    "popovers",
+    "poppers",
+    "customizable"
+  ],
+  "author": "Federico Zivolo <federico.zivolo@gmail.com>",
+  "license": "MIT",
+  "homepage": "https://fezvrasta.github.com/popper.js",
```

Comment:
Wrong domain, at least should be https://fezvrasta.github.io/popper.js since GitHub dropped github.com for users' page for a long time, and also it's been redirected to another domain again.

https://fezvrasta.github.com/popper.js -> https://fezvrasta.github.io/popper.js -> http://popper.js.org/

Looks like http://popper.js.org/ supports https, so many the final url should be https://popper.js.org/